Java/FizzBuzz.java

```diff
@@ -1,19 +1,28 @@
-public class FizzBuzz
+import java.util.*;

```

Comment:
Hi @ARYAMANSHIVAMJH, Thanks for your contribution. Your file FizzBuzz.java is overwriting one already present file. Please update your file name